countWorkingDays.ts

```diff
@@ -0,0 +1,41 @@
+import dayjs, { Dayjs } from 'dayjs';
+import utc from 'dayjs/plugin/utc';
+import timezone from 'dayjs/plugin/timezone';
+import { holidaysDays } from './holidayTable';
+
+dayjs.extend(utc);
+dayjs.extend(timezone);
+dayjs.tz.setDefault('Europe/Warsaw');
+
+const isWeekendOrHoliday = (date: Dayjs): boolean => {
+    const dayOfWeek = date.day();
+    const dateString = date.format('YYYY-MM-DD');
+
+    const isWeekend = dayOfWeek === 0 || dayOfWeek === 6;
```

Comment:
Jeżeli coś spełnia ten warunek, to nie ma sensu sprawdzać też isHoliday - napisałeś bardzo czytelnie, za co plus, ale możesz też rozważyć optymalizację wydajności kodu w tym miejscu.